datascripts/Misc/Ids.ts

```diff
@@ -927,4 +927,7 @@ export const Ids = {
      * - value is hardcoded in server, see TSSmartScripts.h
      */
     condition_types: new StaticIDGenerator('smart_events',100),
+
+    /** Starts at 202500, highest base value is 202482 */
+    pool_template: new StaticIDGenerator('pool_template', 202500),
```

Comment:
does this really need to be a static id? where would you reference it?

Reply:
You are correct, it doesn't need to be. My misunderstanding, double checked the generators code and understand the difference now. This should be fine as just dynamic.